src/pages/instances/forms/DuplicateInstanceForm.tsx

```diff
@@ -0,0 +1,225 @@
+      ).required(),
+    }),
+    onSubmit: (values) => {
+      createInstance(
+        JSON.stringify({
+          //Does not yet consider targetClusterMember or storagePool
+          description: instance.description,
+          name: values.instanceName,
+          source: {
+            allow_inconsistent: values.allowInconsistent,
+            instance_only: values.instanceOnly,
+            source: instance.name,
+            type: "copy",
+          },
+          stateful: values.stateful,
+        }),
+        instance.project,
+      )
+        .then(() => {
+          notifySuccess(values.instanceName);
+          close();
+        })
+        .catch((e) => {
+          toastNotify.failure(
+            `Instance creation for "${values.instanceName}" failed.`,
+            e,
+          );
+          close();
+        });
+    },
+  });
+
+  return (
+    <Modal
+      close={close}
+      className="duplicate-instances-modal"
+      title="Duplicate Instance"
+      buttonRow={
+        <>
+          <Button
+            appearance="base"
+            className="u-no-margin--bottom"
+            type="button"
+            onClick={close}
+          >
+            Cancel
+          </Button>
+          <ActionButton
+            appearance="positive"
+            className="u-no-margin--bottom"
+            loading={formik.isSubmitting}
+            disabled={
+              !formik.isValid ||
+              projectsLoading ||
+              storagePoolsLoading ||
+              clusterMembersLoading
+            }
+            onClick={() => void formik.submitForm()}
+          >
+            Duplicate and start
+          </ActionButton>
+        </>
+      }
+    >
+      <Form onSubmit={formik.handleSubmit}>
+        <Input
+          {...formik.getFieldProps("instanceName")}
+          type="text"
+          label="New instance name"
+        />
+        {isClustered ?? (
+          <Select
+            {...formik.getFieldProps("targetClusterMember")}
+            id="targetClusterMember"
+            label="Target cluster member"
+            options={clusterMembers.map((clusterMember) => {
+              return {
+                label: clusterMember.server_name,
+                value: clusterMember.server_name,
+              };
+            })}
+          />
+        )}
+        <Select
+          {...formik.getFieldProps("targetStoragePool")}
+          id="storagePool"
+          label="Storage pool"
+          options={storagePools.map((storagePool) => {
+            return {
+              label: storagePool.name,
+              value: storagePool.name,
+            };
+          })}
+        />
+        <Select
+          {...formik.getFieldProps("targetProject")}
+          id="project"
+          label="Target project"
+          options={projects.map((project) => {
+            return {
+              label: project.name,
+              value: project.name,
+            };
+          })}
+        />
```

Comment:
```suggestion
```
I think it is not possible to duplicate an instance into another project. So better remove this input?

Can also remove the initial value, and the projects query above.

Reply:
Saw comment from @piperdeck regarding the project selector. @edlerd according to the [lxc copy](https://documentation.ubuntu.com/lxd/en/latest/reference/manpages/lxc/copy/) docs, it's possible to specify a target project for the copy operation (using the `--target-project` flag). I think we should include the project selector here?